cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/ControlFlowSensitiveDFGPass.kt

```diff
@@ -76,7 +77,7 @@ open class ControlFlowSensitiveDFGPass : Pass() {
      * Removes all the incoming and outgoing DFG edges for each variable declaration in the function
      * [node].
      */
-    private fun clearFlowsOfVariableDeclarations(node: FunctionDeclaration) {
+    private fun clearFlowsOfVariableDeclarations(node: Node) {
```

Comment:
Why now `Node` instead of the `FunctionDeclaration`?  The doc string reads like this should be reverted.

Reply:
I updated the text to fit the change. I changed it to `Node` because probably, we also want to handle blocks of code which aren't in any function but (e.g. a script)